modules/cert-manager-istio-creating-issuer.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-operator-integrating-istio.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cert-manager-istio-creating-issuer_{context}"]
+= Creating the issuer for the {cert-manager-operator} with Istio-CSR
+
+Use this procedure to create the issuer for
```

Comment:
This is missing the second part of the sentence. Unless you already have something in mind, you can probably just repeat what the title is saying:
```suggestion
Use this procedure to create the issuer for the {cert-manager-operator} with Istio-CSR
```